src/components/DisplayProfile.jsx

```diff
@@ -12,7 +12,15 @@ const DisplayProfile = props => {
 
   const userDataGrab = async () => {
     if (props.userAttrs) {
-      props.changeUserShowData(await getUserData(props.userAttrs.id));
+      let response = await getUserData(props.userAttrs.id)
+      if (response === "Request failed with status code 401") {
```

Comment:
Nice. Could use response.data or response.code here instead of the message? just a thought, no blocker.